ads/how-to-guides/dynamic-backdrops.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 8
+---
+
+# Dynamic backdrops
+
+Some layouts, such as L-shape and Double Box, make use of a backdrop. These can be specified on both the monetized stream and the ad break in 2 ways:
+
+- `backdropURI`: Optional URI containing the default backdrop to be used during the Double Box or L-shape ads.
+- `backdropURIGamProperties`: Optional property with configuration values for a dynamic backdrop loaded via GAM to be used during the Double Box or L-shape ads. This property has priority over `backdropURI`.
+
+## Using the OptiView Ads service REST API
+
+The way to specify backdrops is through the OptiView Ads service API when creating or updating both monetised streams and adbreaks.
+
+The body of the post request for both monetised streams and ad breaks can have the following properties:
```

Comment:
Nitpick again sorry here it is also the english spelling instead of `monetized`😛